aip/cloud/2510.md

```diff
@@ -28,48 +28,50 @@ different services use different project identifiers.
 is an [alias][], and behaves as such. Additionally, third-party services are
 unable to _accept_ project IDs.
 
-Externally-facing Google APIs **should** accept both project IDs and project
-numbers for incoming API requests.
-
-However, the project number is the canonical identifier, and therefore APIs
-**must** use project numbers in responses, regardless of whether they were sent
-a project ID or a project number.
-
 The rationale for this is:
 
-- Resource should always have one canonical identifier.
+- Each resource should always have one canonical identifier.
 - Because Google's privacy policy restricts the use of project IDs, both
   internally and with partners, only the project number can be the canonical
   identifier.
 - Translating from an [alias][] to a canonical identifier should be consistent
-  everywhere, including here.
+  everywhere, including project identifiers.
 - Returning resources with project numbers increases the chances that these are
   what users store, which may be important if using third-party services.
 
+### Google APIs
+
+Externally-facing Google APIs **should** accept both project IDs and project
+numbers for incoming API requests.
+
+However, the project number is the canonical identifier, and therefore APIs
+**must** use project numbers in responses, regardless of whether they were sent
+a project ID or a project number.
+
 ### Internal Google services
 
 Internal Google services **must** use project numbers for internal data storage
 and for output. Project identifiers are widely used as storage keys, which
-often appear in logs and metrics. (Project IDs are user-settable and thus
-considered user data, but project numbers are not.)
+often appear in logs and metrics. Project IDs are user-settable and thus
+considered PII and user data, but project numbers are not.
 
 Therefore, when an internal service calls an external Google APIs, it
 **should** use project numbers for making API requests.
 
 ### Third-party services
 
 [Third-party services](https://cloud.google.com/marketplace) that are
-integrated with Google Cloud Platform **must** only store or provide project
+integrated with Google Cloud Platform **must** only use and store project
```

Comment:
Hmm. This implies that if I'm writing an API that consumes a Google API, I can't accept a project ID and pass that on? That seems pretty odd.

Reply:
Yeah, it seems like this should be changed back. The requirement really was "no storing or providing" project IDs, as I understood it.